templates/feature.html

```diff
@@ -82,44 +82,30 @@
           {% else %}
             Enabled by default
           {% endif %}
-        {% endif %} in <b>desktop Chrome {{ feature.meta.milestone_str }}</b>
+        {% endif %}
+        {% if feature.bug_url %}
+        (<a href="{{ feature.bug_url }}">launch bug</a>)
+        {% endif %}
+        in: </br>
+        {% if feature.shipped_milestone %}
+          - <b>Chrome for desktop release {{ feature.meta.milestone_str }}</b></br>
```

Comment:
Yes for sure, add classes as needed (although it may be simpler to just use a selector that relies on the dom structure).